ansible_kind/kind_setup.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- hosts: all
+  gather_facts: no
+  strategy: free
+  user: ubuntu
+  tasks:
+   - name:
+     ansible.builtin.wait_for_connection:
+   - name: Copy multuclusterkind directory
+     copy: 
+      src: ./scripts/multiclusterkind 
+      dest: /home/ubuntu/ 
```

Comment:
Is the assumption that a user named `ubuntu` will be created by terraform scripts? OR would this need to be a parameter passed?

Reply:
It is a ubuntu VM, so a ubuntu user will be there.. if we want to create more users then either the terraform needs to enhanced .